src/msgpack-connection.js

```diff
@@ -15,11 +15,11 @@ class MsgpackConnection {
 			codec: msgpack.createCodec({binarraybuffer: true})
 		}
 		// all the callbacks!
-		this.onOpen = null
-		this.onEnd = null
-		this.onMessage = null
-		this.onClose = null
-		this.onError = null
+		this.onOpen = noop
+		this.onEnd = noop
+		this.onMessage = noop
+		this.onClose = noop
+		this.onError = noop
 	}
 
 	connect(onMessage, onEnd, onOpen, connectionOptions) {
```

Comment:
there is a part of me that really wants `onOpen, onEnd`... but I can think of no justification, at all.